docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -82,34 +82,32 @@ The GUI has no Perl. Update the following files in the *develop branch*:
 > This section is relevant for Zonemaster-Engine, Zonemaster-CLI and
 > Zonemaster-Backend.
 
-If needed update Makefile.PL in the *develop branch*.
+If needed, update Makefile.PL in the *develop branch*. See [Appendix A] for
+how the version number should be specified in `Makefile.PL`.
 
 ### Zonemaster-Engine
 
 In [Zonemaster-Engine Makefile.PL] set the lowest version of Zonemaster-LDNS
-that Zonemaster-Engine requires. If unsure, set the version of Zonemaster-LDNS
+that Zonemaster-Engine requires. Usually, set the version of Zonemaster-LDNS
 included in this release since that is what has been tested. E.g.
 
 ```
-requires 'Zonemaster::LDNS'   => 2.0;
+requires 'Zonemaster::LDNS'   => 2.001;
 ```
 
-### Zonemaster-CLI
+### Zonemaster-CLI and Zonemaster-Backend
 
-In [Zonemaster-CLI Makefile.PL] set the lowest version of Zonemaster-LDNS and
-Zonemaster-Engine that Zonemaster-CLI requires. If unsure, set the versions
-included in this release since that is what has been tested. E.g.
+In [Zonemaster-CLI Makefile.PL] and in [Zonemaster-Backend Makefile.PL],
+respectively, set the lowest version of Zonemaster-LDNS and
+Zonemaster-Engine that Zonemaster-CLI and Zonemaster-Backend, respectively,
+require. Usually, set the versions included in this release since that is
+what has been tested. E.g.
```

Comment:
If we say "usually" here, we should at least give an example of what constitutes an unusual situation. Such an example is when Backend/CLI doesn't use any of the features introduced in the dependency since its last minor version bump.

Reply:
To save us problem we should steer the users to install versions that constitute a Zonemaster product version, because that is what we have tested. If in a new product version includes updated Engine but the same CLI as the previous version, then we do not update what version CLI will require.